documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -260,7 +323,23 @@ For the same _ReferenceElement,_ the payload is minimized to the following in ca
 }
 ----
 
-For a _File_ named “Document”, the payload is minimized to the following:
+If the _Reference_ itself is the subject for ValueOnly the serialization is minimized to the following:
+[source,json,linenums]
+
+[source,json,linenums]
```

Comment:
duplicate of line 327? I am not an expert on AsciiDoc but it looks like an error.